uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Component/UserInformationLinkItem.php

```diff
@@ -35,7 +35,7 @@ public function __construct(CMSUserPath $cmsUserPath, $contactData) {
    */
   public function render() {
     $userInformationMarkup = '
-      <p><span class="crm_contact_action_menu__bold_text">User: </span> 
+      <p><span class="crm_contact-actions--bold">User: </span> 
```

Comment:
what's this class for? Please add screenshot

Reply:
@debarshi suggested not to use `<strong></strong>`. So, this class is created instead to add `font-weight:600`. 
<img width="355" alt="screen shot 2018-01-24 at 2 01 32 pm" src="https://user-images.githubusercontent.com/6307362/35321474-234bb6a4-010f-11e8-9d8f-de0bf23d4fe7.png">

Here 'User' and 'Roles' test is made bold suing the class.

Reply:
that looks like a definition list (`<dl>`) to me. Shoreditch provides plenty of variations of definition lists, please check the style guide (/civicrm/styleguide/bootstrap) (i think this one is the "Horizontal Inline" variation)

cc @deb1990

Reply:
Thanks! for the suggestion. I have checked and agree its more appropriate to use `<dl>` in this case.